pre-build.mjs

```diff
@@ -0,0 +1,4 @@
+import { assets } from '@lukso/web-components/tools/assets';
```

Comment:
maybe put a comment here just to describe something like this:
// This pre-build script is responsible to copy assets and from lukso web-components so they would work with the correct styles and images that are used on the components. @dzbo what do you think?

Reply:
Yes for sure this should be commented. By the way, almost "every line" of this example app should have some sort of explanation what is happening so users can understand and learn.